data/lang/module-combat/en.json

```diff
@@ -225,7 +225,7 @@
   },
   "MILITARY_LAND_THERE": {
     "description": "",
-    "message": "Visit a station in the territory protected by the {org}!"
+    "message": "Visit a station in the territory protected by the {org}! This must be done before the due date! Otherwise the entire operation will fail."
```

Comment:
Combine the sentences to one, perhaps something like: "The operation requires targets be eliminated by date" 
(assuming my guesstimation of mission flavor fits)